custom_components/loxone/api.py

```diff
@@ -839,10 +852,10 @@ def read_user_salt_responce(self, reponse):
         value = js['LL']['value']
         self.key = value['key']
         self.salt = value['salt']
-        self.hash_alg = value.get("hashAlg", "SHA256")
+        self.hash_alg = value.get("hashAlg", "SHA1")
```

Comment:
Changed the default here to SHA1 on basis that if the server doesn't return a value for `hashAlg` it must be older than v11